src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...args) {
+  return (...values) => {
```

Comment:
good, but you can write simply
```suggestion
  return (...values) => callback(...arg, ... values);
```